test/mocks/MockTing.scala

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This class is something I did during my initial spike - appears to still be knocking around.  If its used, can we name it something else? :joy:

Reply:
We not using this, I will delete....